encoder/vaapiencoder_vp8.cpp

```diff
@@ -75,6 +75,10 @@ void VaapiEncoderVP8::resetParams()
 {
     //5 times compress ratio
     m_maxCodedbufSize = width() * height() * 3 / 2;
+    //when the resolution of video is very small such as 96x96 and buffer plus the header of vp8,
+    //the size of actual output buffer maybe be bigger the m_maxCodedbufSize.
+    //So plusing 4096 avoids the situation.
+    m_maxCodedbufSize += 0x1000;
```

Comment:
Hi Daniel:
The m_maxCodedbufSize is just a guessed value. plus 4k should be a problem for large resolution,  Since it's already very big.
We do not know the compression ratio in driver, So we use entire frame size previously, But Linhai found for small resolution the header size maybe very large. So he plus 4k to make sure it's not overflow